build.gradle

```diff
@@ -3,8 +3,8 @@
 buildscript {
     ext.kotlin_version = '1.3.71'
     ext.jna_version = '5.6.0'
-    ext.android_gradle_plugin_version = '3.6.0'
-    ext.android_components_version = '56.0.0'
+    ext.android_gradle_plugin_version = '4.0.1'
+    ext.android_components_version = '63.0.0'
```

Comment:
For compatibility with the version of glean-gradle-plugin that is in the submodule.